2021-10-21 00:00:49,388 INFO    MainThread:46449 [wandb_setup.py:_flush():69] setting env: {'project': 'exercise_14', 'run_group': 'dev'}
2021-10-21 00:00:49,388 INFO    MainThread:46449 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-21 00:00:49,388 INFO    MainThread:46449 [wandb_init.py:_log_setup():342] Logging user logs to /Users/obara/nd0821-c2-build-model-workflow-exercises/lesson-5-final-pipeline-release-and-deploy/exercises/exercise_14/starter/preprocess/wandb/run-20211021_000049-st4939zy/logs/debug.log
2021-10-21 00:00:49,389 INFO    MainThread:46449 [wandb_init.py:_log_setup():343] Logging internal logs to /Users/obara/nd0821-c2-build-model-workflow-exercises/lesson-5-final-pipeline-release-and-deploy/exercises/exercise_14/starter/preprocess/wandb/run-20211021_000049-st4939zy/logs/debug-internal.log
2021-10-21 00:00:49,389 INFO    MainThread:46449 [wandb_init.py:init():375] calling init triggers
2021-10-21 00:00:49,389 INFO    MainThread:46449 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-10-21 00:00:49,389 INFO    MainThread:46449 [wandb_init.py:init():424] starting backend
2021-10-21 00:00:49,389 INFO    MainThread:46449 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2021-10-21 00:00:49,696 INFO    MainThread:46449 [backend.py:ensure_launched():135] starting backend process...
2021-10-21 00:00:49,711 INFO    MainThread:46449 [backend.py:ensure_launched():139] started backend process with pid: 47169
2021-10-21 00:00:49,712 INFO    MainThread:46449 [wandb_init.py:init():429] backend started and connected
2021-10-21 00:00:49,717 INFO    MainThread:46449 [wandb_init.py:init():477] updated telemetry
2021-10-21 00:00:49,718 INFO    MainThread:46449 [wandb_init.py:init():500] communicating current version
2021-10-21 00:00:50,141 INFO    MainThread:46449 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.5 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-21 00:00:50,141 INFO    MainThread:46449 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-10-21 00:00:50,404 INFO    MainThread:46449 [wandb_init.py:init():540] starting run threads in backend
2021-10-21 00:00:50,924 INFO    MainThread:46449 [wandb_run.py:_console_start():1601] atexit reg
2021-10-21 00:00:50,924 INFO    MainThread:46449 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.REDIRECT
2021-10-21 00:00:50,925 INFO    MainThread:46449 [wandb_run.py:_redirect():1480] Redirecting console.
2021-10-21 00:00:50,926 INFO    MainThread:46449 [wandb_run.py:_redirect():1536] Redirects installed.
2021-10-21 00:00:50,926 INFO    MainThread:46449 [wandb_init.py:init():565] run started, returning control to user process
2021-10-21 00:00:51,129 INFO    MainThread:46449 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 1
2021-10-21 00:00:51,129 INFO    MainThread:46449 [wandb_run.py:_restore():1543] restore
2021-10-21 00:00:53,460 INFO    MainThread:46449 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1217
  total_bytes: 1217
}

2021-10-21 00:00:55,239 INFO    MainThread:46449 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1217
  total_bytes: 1217
}

2021-10-21 00:00:55,458 INFO    MainThread:46449 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1217
  total_bytes: 1217
}

2021-10-21 00:00:56,449 INFO    MainThread:46449 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1217
  total_bytes: 10510
}

2021-10-21 00:00:56,555 INFO    MainThread:46449 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1217
  total_bytes: 10510
}

2021-10-21 00:00:56,663 INFO    MainThread:46449 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1217
  total_bytes: 10510
}

2021-10-21 00:00:56,769 INFO    MainThread:46449 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10510
  total_bytes: 10510
}

2021-10-21 00:00:56,875 INFO    MainThread:46449 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10510
  total_bytes: 10510
}

2021-10-21 00:00:56,980 INFO    MainThread:46449 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10510
  total_bytes: 10510
}

2021-10-21 00:00:57,086 INFO    MainThread:46449 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10510
  total_bytes: 10510
}

2021-10-21 00:00:57,192 INFO    MainThread:46449 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10510
  total_bytes: 10510
}

2021-10-21 00:00:57,862 INFO    MainThread:46449 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10510
  total_bytes: 10510
}

2021-10-21 00:00:57,969 INFO    MainThread:46449 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10510
  total_bytes: 10510
}

2021-10-21 00:00:59,074 INFO    MainThread:46449 [wandb_run.py:_show_files():1915] logging synced files
